meta/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+galaxy_info:
+  role_name: dkim
+  author: Tomáš Jacík
+  company: sunfoxcz
+  description: Configure OpenDKIM with Postfix
+  min_ansible_version: 2.4
```

Comment:
Will this really work with 2.4? Did you tested it?

Reply:
Nope.... you right. Sorry, copy/paste from  another role. Better to put the tested version, nobody will be running 2.4.

Reply:
Then I propose current `2.9` version.

Reply:
Ok, It's what I put.